docs/UserGuide.md

```diff
@@ -279,7 +279,7 @@ Updates the amount of tuition fee paid by the specified student after a lesson.
 
 ### Recording unpaid tuition fee of a student: `owe`
 
-Updates the amount of tuition fee owed by a specified student after a lesson.
+Updates the amount of tuition fee owed by a specified student after a lesson. You should use the [`settle` command](#settling-outstanding-fees-from-students-settle) to settle the tuition fee owed by the student after he/she pay for the outstanding fees.
```

Comment:
Not sure if we need to tell about settle in owe since functionalities are very different

Reply:
I was thinking like settle updates both owe and pay. Thats why I placed this line for owe also

Reply:
Alright then makes sense.